public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -478,6 +483,32 @@ public ConceptAnalysis apply(ConceptAnalysis ca) {
             }
 
             List<AchillesAnalysis> analyses = achillesAnalysisDao.findSurveyAnalysisResults(surveyConceptId, qlist);
+            for(AchillesAnalysis analysis: analyses){
```

Comment:
nit: please use consistent spacing throughout this file, specifically most of the codebase uses
```
for (Foo f : flist) {
...
Foo f = 1;
if (true) {
...
```

Reply:
oops. This file shouldn't even be in the pr. I made no intentional changes here.  I put in the version from master.